RELEASES.md

```diff
@@ -96,38 +96,58 @@ deadline of 3 weeks.
 
+#### In the week prior to the release
+
+* Check the [release milestone](https://github.com/envoyproxy/envoy/milestones).
 * Take a look at open issues tagged with the current release, by
   [searching](https://github.com/envoyproxy/envoy/issues) for
   "is:open is:issue milestone:[current milestone]" and either hold off until
   they are fixed or bump them to the next milestone.
-* Begin marshalling the ongoing PR flow in this repo. Ask maintainers to hold off merging any
-  particularly risky PRs until after the release is tagged. This is because we aim for main to be
-  at release candidate quality at all times.
+* Begin marshalling the ongoing PR flow in this repo.
+* Ask maintainers to hold off merging any particularly risky PRs until after the release is tagged.
+* Conversely, encourage maintainers to land anything that should be included.
 * Do a final check of the [release notes](changelogs/current.yaml):
   * Make any needed corrections (grammar, punctuation, formatting, etc.).
   * Check to see if any security/stable version release notes are duplicated in
     the major version release notes. These should not be duplicated.
-  * Switch the repo to "release" mode by running `bazel run @envoy_repo//:release`. This tool
-    will create a commit with the necessary changes for a release.
-  * Update the [RELEASES](RELEASES.md) doc with the relevant dates. Now, or after you cut the
-    release, please also make sure there's a stable maintainer signed up for next quarter,
-    and the deadline for the next release is documented in the release schedule.
-  * Get a review and merge.
-* Create a pull request with that commit and **wait for tests to pass**.
-* Once the tests have passed, and the PR has landed, CI will automatically create the tagged release and corresponding release branch.
-* Switch the repo back to "dev" mode by running `bazel run @envoy_repo//:dev`. This tool will create a commit with the
-  necessary changes to continue development.
-* Create a pull request with that commit.
-* Run the deprecate_versions.py script (`bazel run //tools/deprecate_version:deprecate_version`)
+* Make sure there's a stable maintainer signed up for next quarter.
+* Select a date for the next scheduled release.
+
+#### The day before release
+
+* Create a release PR running the Github workflow `create-release` from:
+
+    https://github.com/envoyproxy/envoy/actions/workflows/envoy-release.yml
+
+  This will warm the caches and ensure the workflow CI can run correctly.
+* Create a summary by updating the markdown in the [release summary](changelogs/summary.md).
+* Re-check the steps [above](#in-the-week-prior-to-the-release)
+* Clear the milestone as far as possible, either by finding resolutions or removing issues/PRs.
+
+#### On the day of the release
+
+* Re-create the release PR and lock the branch.
+* If necessary, make any last minute adjustments to the summary in the PR. The commit message will be used to create the release.
```

Comment:
this isnt always necessary - yday i spotted a couple of last minute typos so rather than rebasing the auto generated commit i squashed an amended commit - the point here tho is that whatever the commit shows will be posted as the release message (which is editable fwiw)

Reply:
~relatedly its supposed to automatically add signoff for last committer to changelog/summary.md but that doesnt appear to be working atm

Reply:
I would like to avoid that as much as possible...
The assumption should be that the summary was approved previously, and we should live with that.
In other words, one might need to deviate a bit from the current checklist, but I don't think this is something we want to call out.

Reply:
not sure here - i think we at least want to emphasize that the commit is used for the release info